js/packages/teams-ai/src/StreamingResponse.ts

```diff
@@ -113,6 +114,22 @@ export class StreamingResponse {
         return this._queueSync!;
     }
 
+    /**
+     * Sets the attachments to attach to the final chunk.
+     * @param attachments List of attachments.
+     */
+    public setAttachments(attachments: Attachment[]): void {
```

Comment:
This implies that attachments is never undefined, is that correct?

Reply:
if they are sending attachments, then yes